openshift/pfioh-openshift-template-without-swift.json

```diff
@@ -86,6 +94,12 @@
               "hostPath": {
                 "path": "/tmp/share"
               }
+            },
+            {
+              "name": "auth-config",
+              "secret":{
+                "secretName": "config.cfg"
```

Comment:
@betaredex Name here is inconsistent as well.  Can you open a new pull with @iamemilio's changes and make the necessary adjustments.